db/migrate/20230712192044_add_distribution_engine_to_products.rb

```diff
@@ -0,0 +1,7 @@
+class AddDistributionEngineToProducts < ActiveRecord::Migration[7.0]
+  def change
+    add_column :products, :distribution_engine, :string, null: true
```

Comment:
Should we add `engine` to releases and artifacts? May be easier to just leave it alone for now.

Reply:
Right now, it doesn't make sense to change an engine once set, so let's just leave it as-is right now. You can't license multiple engines under the same product yet, anyways, so we can revisit.